### PR TITLE
Item

### PR DESCRIPTION
**Pull request by Pocketzs**
Completed Item class and Item repository class

### CODE REVIEW

**Comment by KaraJoHo** [comment]
Nice job!

**Review by Kledin85 — CHANGES_REQUESTED** [review]
acknowledge some hounds comments.

File: spec/item_repository_spec.rb [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        :merchant_id => 2
+      })
+      ir.items << i1
+      
+      ir.delete(1)
+
+      expect(ir.items).to eq []
+    end
+  end
+end
```
[the remark this answers is not in the document]
Comment (beddings81) [reply spec/item_repository_spec.rb:331]: add new line here

**Review by beddings81 — CHANGES_REQUESTED** [review]
great job! Just need a few syntax changes

**Review by KaraJoHo — APPROVED** [review]

**Review by Kledin85 — APPROVED** [review]

**Review by beddings81 — APPROVED** [review]

**Pull request merged**